core/domain/collection_services_test.py

```diff
@@ -835,7 +841,10 @@ def test_retrieval_of_multiple_collections(self):
 
         self.assertNotIn('doesnt_exist', result)
 
-        with self.assertRaises(Exception):
+        with self.assertRaisesRegexp(
+            Exception,
+            '\'module\' object has no attribute '
+            '\'get_multiple_collection_by_id\''):
```

Comment:
Ditto.

Reply:
@Hudda I don't see this fixed, can you please verify before assigning reviewers!

/cc @bansalnitish @ankita240796

Reply:
Look again and please try to see the file carefully before leaving a comment because this increases the time of the review process. (I am saying this because it is not the first time this is happening.) And in the GSoC, we have a very short time available.

Reply:
@Hudda, Sorry I was looking through the filename link of the comment thread and ended up looking through an old commit changes, I hope you understand. I'll make sure to check the commit id in the link before commenting, I think that will help me to avoid this problem and I won't repeat it! :) 

>  this increases the time of the review process. (I am saying this because it is not the first time this is happening.)

Also, I try my best to unblock you asap, sorry if have blocked you before! (I'll try my best to review PRs within 24 hrs. and unblock PRs!)